core-web/libs/edit-content/src/lib/fields/dot-edit-content-radio-field/dot-edit-content-radio-field.component.spec.ts

```diff
@@ -0,0 +1,167 @@
+import { Spectator, createComponentFactory } from '@ngneat/spectator';
+
+import { ControlContainer, FormGroupDirective } from '@angular/forms';
+
+import { DotEditContentRadioFieldComponent } from './dot-edit-content-radio-field.component';
+
+import {
+    RADIO_FIELD_BOOLEAN_MOCK,
+    RADIO_FIELD_FLOAT_MOCK,
+    RADIO_FIELD_INTEGER_MOCK,
+    RADIO_FIELD_TEXT_MOCK,
+    createFormGroupDirectiveMock
+} from '../../utils/mocks';
+
+describe('DotEditContentRadioFieldComponent', () => {
+    let spectator: Spectator<DotEditContentRadioFieldComponent>;
+
+    const createComponent = createComponentFactory({
+        component: DotEditContentRadioFieldComponent,
+        componentViewProviders: [
+            { provide: ControlContainer, useValue: createFormGroupDirectiveMock() }
+        ],
+        providers: [FormGroupDirective],
+        detectChanges: false
+    });
+
+    beforeEach(() => {
+        spectator = createComponent();
+    });
+    describe('DotEditContentRadioFieldComponent with Text DataType', () => {
+        it('should have a options array as radio with Text dataType', () => {
+            const expectedList = [
+                {
+                    label: 'Uno',
+                    value: 'uno'
+                },
+                {
+                    label: 'Dos',
+                    value: 'dos'
+                }
+            ];
+            spectator.setInput('field', RADIO_FIELD_TEXT_MOCK);
+            spectator.detectChanges();
+            expect(spectator.component.options).toEqual(expectedList);
```

Comment:
We should be testing the DOM like:

```suggestion
            expect(spectator.query(Dropdown).options).toEqual(expectedList);
```